packages/types/src/pricing/common/price-rule.ts

```diff
@@ -26,6 +57,21 @@ export interface CreatePriceRuleDTO {
   price_list_id: string
 }
 
+/**
+ * 
+ * @interface
+ * 
+ * An object used to specify the necessary data to update a price rule.
+ * 
+ * @prop id - A string indicating the ID of the price rule to update.
+ * @prop price_set_id - A string indicating the ID of the associated price set.
+ * @prop rule_type_id - A string indicating the ID of the associated rule type.
+ * @prop is_dynamic - A boolean indicating whether the price rule is dynamic.
```

Comment:
This isn't implemented yet, should we maybe add a note? or remove entirely from TS

Reply:
Sure, I've removed the prop from the TSDocs and added the `@ignore` TSDoc directly on it with a private note that it's not implemented yet. Let me know about other properties/methods/etc... that we should remove from the generated docs too.

Reply:
Rest look great! :)